src/dialog-settings.ts

```diff
@@ -61,6 +62,13 @@ export interface DialogSettings {
    */
   keyEvent?: KeyEventType;
 
+  /**
+   * Determines which type of mouseevent should be used for closing the dialog
+   *
+   * Default: click
+   */
+  mouseEventType?: MouseEventType;
```

Comment:
I think we would want to align this property name with the existing key event property.
I'd suggest either `mouseEvent`. If we want to go further, to describe what it really is, we can do: `closeDialogMouseEvent`

Reply:
for ppl like me who are not completly into this. is it also possible to have so that you cannot close it at all with mouse event. only cancel og ok button